src/Yaapii.Atoms/Map/Joined.cs

```diff
@@ -51,46 +53,59 @@ public sealed class Joined : Map.Envelope
         /// Joined map.
         /// </summary>
         public Joined(params IDictionary<string, string>[] dicts) : this(
-            new Many.Live<IDictionary<string, string>>(dicts)
+            false,
+            dicts
         )
         { }
 
         /// <summary>
         /// Joined map.
         /// </summary>
-        public Joined(IEnumerable<IDictionary<string, string>> dicts) : base(
+        public Joined(bool live, params IDictionary<string, string>[] dicts) : this(
+            new Many.Live<IDictionary<string, string>>(dicts),
+            live
+        )
+        { }
+
+        /// <summary>
+        /// Joined map.
+        /// </summary>
+        public Joined(IEnumerable<IDictionary<string, string>> dicts, bool live = false) : base(
             () =>
-            new LazyDict(
+            new Map.Live(
                 new Enumerable.Joined<IKvp>(
-                    new Mapped<IDictionary<string, string>, IEnumerable<IKvp>>(dict => 
-                        new Mapped<KeyValuePair<string, string>, IKvp>(entry => 
+                    new Mapped<IDictionary<string, string>, IEnumerable<IKvp>>(dict =>
+                        new Mapped<KeyValuePair<string, string>, IKvp>(entry =>
                             new Kvp.Of(entry.Key, entry.Value),
                                 dict
                             ),
                             dicts
                     )
                 )
-            )
+            ),
+            live
         )
         { }
 
         /// <summary>
         /// A live joined map.
         /// </summary>
-        public sealed class Live : Map.LiveEnvelope
+        public sealed class Live : Map.Envelope
```

Comment:
This class should be removed - see text area. It was an idea having a „live“ along with the by default sticky object, but this idea is replaced by the boolean parameter which we introduced recently, and which is added in this branch.